### DISCUSSION TITLE
Is there a way to get the tests running on the CodeSandbox example?

### DISCUSSION THREAD

**Discussion by cmacdonnacha** [mswjs/msw#1466] [General]
I have forked the [official codesandbox example](https://codesandbox.io/s/msw-react-xx1c8) and changed the codesandbox `view` to `test` but they never seem to run. Just wondering if anyone else has been able to get them to run?

See my forked version: https://codesandbox.io/s/jolly-johnson-filq6l?file=/sandbox.config.json

![image](https://user-images.githubusercontent.com/9777763/202317970-9ba87550-d487-49a8-9edd-41f4debf7ccb.png)

**Comment by kettanaito** [comment]
When I try to run tests manually in the terminal via `yarn test` I get:

```
No tests found, exiting with code 0
```

It looks like `react-scripts` don't detect the `test/App.test.js` file at all. Were there any changes that now require a different folder/file structure in this regard?

**Reply by cmacdonnacha** [reply]
I think if you move the file into the `src` folder it'll at least find them, still fails for me though. 

![image](https://user-images.githubusercontent.com/9777763/202455841-cc3b462e-86f4-40f1-a4a3-ea1b7cfef5c6.png)

I wonder how the "Tests" tab on CodeSandbox works under the hood? Does it just run `yarn test` for us?

**Reply by kettanaito** [reply]
Yes, I believe it detects the `test` script in package.json and executes that. If there's none, it may execute Jest (or other runner) by default, although I'm not sure about this one. I haven't verified the automated tests in our sandbox for a while, things might've changed.

**Reply by cmacdonnacha** [reply]
So `yarn test` works in the terminal after:

1. Moving the test file under `src`.
2. Removing [this line](https://codesandbox.io/s/msw-react-xx1c8?file=/src/setupTests.js:247-282) from the official template. I'm guessing that is there to allow tests to run inside the "tests" tab which runs on the browser. 

It would be nice to get these tests running inside that tests tab, but at least it works in the terminal. Thanks for your help. 👍
